person.rb

```diff
@@ -25,5 +26,9 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
+
```

Comment:
- [ ] We need to pass in `date` and `book`, not `person`.

- [ ] We need to also rearrange the arguments here to match the Rental class. In the end we should have something like this:

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```